### ISSUE TITLE
CSA Live Reload

### ISSUE THREAD

**Issue by hasanirogers** [pikapkg/create-snowpack-app#43]
I've been using Snowpack to create a Lit-Element app and it's great. However live reloads only works for files modified directly in the /src directory. I have folders in there for each element and live reload does not work when I modify them. A normal reload doesn't work either. I have to restart the dev server to see changes. 

How can I fix this? Here's my snowpack config:

```
{
  "scripts": {
    "mount:public": "mount public --to .",
    "mount:web_modules": "mount web_modules",
    "mount:src": "mount src --to _dist_",
    "build:js": "cat"
  },
  "installOptions": {},
  "devOptions": {}
}
```

**Comment by FredKSchott** [comment]
I can't reproduce this on master, it may have been solved with the latest chokidar improvements. Mind updating your version of snowpack and then trying again? Also, it would be helpful to know if you were on Windows/Mac/Linux.

**Comment by hasanirogers** [comment]
I'm using the latest 2.0 version of snowpack (`"snowpack": "^2.0.0-0"`) FYI. Also I'm on Ubuntu 18.04.4 LTS.

node: v10.20.1
npm: v6.14.4

**Comment by FredKSchott** [comment]
This should be fixed now, we now watch all mounted directories:

```js
  const watcher = chokidar.watch(
    mountedDirectories.map(([dirDisk]) => dirDisk),
    {
      ignored: config.exclude,
      persistent: true,
      ignoreInitial: true,
      disableGlobbing: false,
    },
  );
```

**Comment by moretti** [comment]
@FredKSchott I'm still experiencing the same problem (`snowpack@2.5.4`), I created a project to reproduce the issue:
https://github.com/moretti/snowpack2-mono-repo

In this particular case there are few modules in the mono repo that are automatically transpiled by the TypeScript compiler, e.g. `packages/package-a/src/index.ts` ->  `packages/package-a/dist/index.js`.
I noticed that snowpack can detect a file change when a file on the `dist` folder (e.g. `packages/package-b/dist/index.js`) is edited from a text editor, but not when the same file is updated by `tsc` 🤔
